variables.tf

```diff
@@ -1 +1,94 @@
-# Variables placeholder
+variable "protect_s3_bucket_resources" {
+  description = "S3 bucket resource ARNs to protect from bucket and object deletion"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "protect_iam_role_resources" {
+  description = "IAM role resource ARNs to protect from modification and deletion"
+  type        = list(string)
+  default     = []
+}
+
+variable "allowed_regions" {
+  description = "AWS Regions allowed for use (for use with the restrict regions SCP)"
+  type        = list(string)
+  default     = [""]
+}
+
+variable "target" {
+  description = "id of OU or account on which to attach SCP"
+  type        = string
+}
+
+variable "add_deny_root_account" {
+  description = "Add DenyRootAccount to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_deny_leaving_orgs" {
+  description = "Add DenyLeavingOrgs to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_deny_creating_iam_users" {
+  description = "Add DenyCreatingIAMUsers to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_deny_deleting_kms_keys" {
+  description = "Add DenyDeletingKMSKeys to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_deny_deleting_route53_zones" {
+  description = "Add DenyDeletingRoute53Zones to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_deny_all_access" {
+  description = "Add DenyAllAccess to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_require_s3_encryption" {
+  description = "Add RequireS3Encryption to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_deny_deleting_cloudwatch_logs" {
+  description = "Add DenyDeletingCloudwatchLogs to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_protect_s3_buckets" {
+  description = "Add ProtectS3Buckets to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_protect_iam_roles" {
+  description = "Add ProtectIAMRoles to the OU policy."
+  default     = false
+  type        = bool
+}
+
+variable "add_limit_regions" {
+  description = "Add LimitRegions to the OU policy."
+  default     = false
+  type        = bool
+}
+
+# variable "add_all_policies" {
+#   description = "Add all policies in this module to the OU policy."
+#   default     = false
+#   type        = bool
+# }
```

Comment:
I'd recommend just removing this. The cases where this would be set are miniscule.